content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx

```diff
@@ -5,9 +5,8 @@ metaDescription: Prisma Client features that are currently in preview.
 
 <TopBlock>
```

Comment:
I think it would be worth to add a headline a la "Currently active Preview Features" so that the actual content also gets a link in the right "On this page" sidebar navigation.